install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -2638,18 +2638,31 @@ sctp:
 
 auth:
   mTLS:
-    # -- Enable mtls-spiffe authentication method in CiliumNetworkPolicy
-    enabled: false
-    # -- SPIRE server endpoint
-    # This endpoint will be automatically injected later once embedded SPIRE installation is done.
-    spireServerAddress: spire-server.spire.svc.cluster.local:8081
-    # -- SPIRE server connection timeout
-    spireServerConnectionTimeout: 10s
-    # -- SPIRE agent socket path where the SPIRE agent is listening
-    spireAgentSocketPath: /run/spire/sockets/agent/agent.sock
-    # -- SPIRE socket path where the SPIRE delegated api agent is listening
-    spireAdminSocketPath: /run/spire/sockets/admin.sock
-    # -- SPIFFE trust domain to use for fetching certificates
-    spiffeTrustDomain: spiffe.cilium.io
-    # -- port on the agent which is used to mTLS handshakes on
+    # -- Port on the agent which is used to mTLS handshakes on
     port: 4250
+    # Settings for SPIRE
+    spire:
+      # -- Enable SPIRE integration
+      enabled: false
+      # -- Settings to control the SPIRE installation and configuration
+      install:
+        # -- Enable SPIRE installation
+        # Note that this will only take effect if auth.mTLS.enabled is true
+        enabled: false
```

Comment:
don't we want to default this to true so only 1 enable flag is needed? Makes it easy to use

Reply:
however the 2nd install flag will only be triggered when the spire auth is enabled